app/controllers/observations_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ObservationsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_proposal
+  before_action ->{authorize @proposal, :can_edit!}
+
```

Comment:
I think this is missing a `rescue_from` for authorization errors? It's part of a mixin in the ProposalsController